plugins/modules/import_eks.py

```diff
@@ -136,8 +137,10 @@ def execute_module(module):
     addons = module.params['addons']
     wait = module.params['wait'] # if key exists and no value and watch out for wait package in python
     timeout = module.params['timeout']
-    aws_access_key = module.params['aws_access_key']
-    aws_secret_key = module.params['aws_secret_key']
+
+    region, ec2_url, aws_connect_kwargs = get_aws_connection_info(module, boto3=True)
```

Comment:
do we need ec2_url? and region is also not used?

Reply:
removed